### DISCUSSION TITLE
Fibaro FGT001

### DISCUSSION THREAD

**Discussion by TheCopperDragon** [zwave-js/zwave-js#4454] [Q&A]
Hi, 

Hoping someone can help me. I have a Fibaro FGT001 heat controller connected via Z-Wave JS to Home Assistant. 
I can control the valve as expected but I would like to read back the read only parameters as entities. Is this possible and if so, can anyone point me in the right direction please?

![image](https://user-images.githubusercontent.com/65547025/162403123-e9141640-3d37-4b62-9d86-0b1fd258e652.png)

**Comment by AlCalzone** [comment]
Your best bet would be asking this in one of the HomeAssistant channels, e.g. on Discord. I'm not familiar with HA myself and those who are might not follow the discussions here.
